src/briefcase/integrations/docker.py

```diff
@@ -209,27 +209,31 @@ def prepare(self):
             except AttributeError:
                 system_requires = ""
 
-            self._subprocess.run(
-                [
-                    "docker",
-                    "build",
-                    "--tag",
-                    self.command.docker_image_tag(self.app),
-                    "--file",
-                    self.command.bundle_path(self.app) / "Dockerfile",
-                    "--build-arg",
-                    f"PY_VERSION={self.command.python_version_tag}",
-                    "--build-arg",
-                    f"SYSTEM_REQUIRES={system_requires}",
-                    "--build-arg",
-                    f"HOST_UID={self.command.os.getuid()}",
-                    "--build-arg",
-                    f"HOST_GID={self.command.os.getgid()}",
-                    Path(self.command.base_path, *self.app.sources[0].split("/")[:-1]),
-                ],
-                check=True,
-            )
-
+            with self.command.input.wait_bar("Building container..."):
+                self._subprocess.run(
+                    [
+                        "docker",
+                        "build",
+                        "--progress",
+                        "plain",
+                        "--tag",
+                        self.command.docker_image_tag(self.app),
+                        "--file",
+                        self.command.bundle_path(self.app) / "Dockerfile",
+                        "--build-arg",
+                        f"PY_VERSION={self.command.python_version_tag}",
+                        "--build-arg",
+                        f"SYSTEM_REQUIRES={system_requires}",
+                        "--build-arg",
+                        f"HOST_UID={self.command.os.getuid()}",
+                        "--build-arg",
+                        f"HOST_GID={self.command.os.getgid()}",
+                        Path(
+                            self.command.base_path, *self.app.sources[0].split("/")[:-1]
```

Comment:
Another minor formatting thing - add a comma on the end to force black into splitting this line. Generally. the "contents of brackets on a line by itself" is a flag to me that we should be splitting arguments onto separate lines.

Reply:
Updated.